default.config.cfg

```diff
@@ -1,28 +1,35 @@
-# Emoncms Export / Import scripts config file
+# Emoncms Export and Import scripts config file
 
 ## NON-EMONPI DEFAULT CONFIG ##
 
-# These settings are used for both the export & import scripts.
+# These settings are used for both the export and import scripts.
 # To export from one location and import into another
+# Used by both PHP and Bash and comments are not ignored
 
+# CHANGE: USER to your username
+
+# user
+user="USER"
+
+backup_script_location="/home/USER/backup"
```

Comment:
Which folder is this?  The toplevel backup or the backup/backup - not completely clear.